Dart/src/com/jetbrains/lang/dart/ide/toolingDaemon/DartToolingDaemonService.kt

```diff
@@ -105,6 +106,24 @@ class DartToolingDaemonService private constructor(private val project: Project)
     }
   }
 
+  fun registerServiceMethod(service: String, method: String, consumer: DartToolingDaemonServiceConsumer) {
+    val params = JsonObject()
+    params.addProperty("service", service)
+    params.addProperty("method", method)
+    sendRequest("registerService", params, false) {response ->
+      val result = response.getAsJsonObject("result")
+      if (result == null) {
+        logger.error("No result from attempt to register service $service.$method", service, method)
```

Comment:
I removed the second and the third arguments because both `$service` and `$method` are already in the message (the first argument)